packages/admin/dashboard/src/routes/orders/order-detail/components/order-activity-section/order-timeline.tsx

```diff
@@ -840,21 +868,24 @@ const OrderEditBody = ({
   )
 }
 
+/**
+ * Returns count of added and removed item quantity
+ */
 function countItemsChange(
```

Comment:
nit: This section seem somewhat unsafe with all the "!" and type casting. If these are always present and always numbers then it would seem like the types are wrong?

Reply:
These fields will be present for RMA changes, but in theory details of the order change action could have different shapes depending on the concrete operation. But yes, we could add more concrete types for RMA changes and actions we know.